ETC.tsv

```diff
@@ -11766,7 +11766,7 @@ ETC_20150714_011767	Yes!
 ETC_20150714_011768	Control the Magic Generating Stones to stop them all!
 ETC_20150714_011769	Magic Generating Stones
 ETC_20150714_011770	Archon has been released and abducted Jane!{nl}Defeat Archon and save Jane!
-ETC_20150714_011771	The Magic Stone of Interaction
+ETC_20150714_011771	The Magic Stone of Sympathy
```

Comment:
It seems to be a quest about releasing Jane's Spirit from an Archon. The Magic Stone seems to be binding the spirit to the monster thus the spirit cannot be freed unless you stop the magic coming out of the Magic Stone (and defeat Archon later).

I guess Interaction works, but it seems to be a bland word for "magic that binds the two together". So I was thinking either Sympathy or Mutuality (to make it seem like both are being affected by a certain condition).